nifi-stateless/nifi-stateless-bundle/nifi-stateless-engine/src/main/java/org/apache/nifi/stateless/engine/StatelessEngine.java

```diff
@@ -70,4 +72,6 @@
     FlowFileEventRepository getFlowFileEventRepository();
 
     CounterRepository getCounterRepository();
+
+    Duration getStatusTaskSchedule();
```

Comment:
Would argue this isn't really returning a schedule, but rather a frequency or a scheduling period - thoughts on calling this getStatusTaskSchedulingPeriod() or getStatusTaskFrequency()?

Reply:
Fair enough -- I like the latter as it's shorter -- though what about getStatusTaskInterval?  This would match the term used in nifi.properties in a few places already.